npm/vite-dev-server/cypress/support/support.js

```diff
@@ -1,3 +1,9 @@
+import { unmount } from '@cypress/react'
+
 before(() => {
   window.supportFileWasLoaded = true
 })
+
+beforeEach(() => {
```

Comment:
Not sure that it should live in vite dev server. Probably let's move it to to `@cypress/react` itself

Reply:
I will try it

Reply:
Ok, done, it works

Reply:
Great, works fine